lib/index.ts

```diff
@@ -24,6 +25,7 @@ import {
 export {
   buildDepTree,
   buildDepTreeFromFiles,
+  buildDepGraphFromFiles,
```

Comment:
stupid question:

what is the difference between `buildDepTree` and `buildDepGraph`?

(since npm allows cyclic dependencies, it should always be a graph)

Reply:
Wow this is an old pr 🤪 maybe should close it. Yea Snyk started with a tree where cycles were terminated, and has now moved over to a graph where cycles are possible